advancing-bitcoin/2020/2020-02-06-andrew-poelstra-miniscript-intro.es.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Andrew Poelstra - Miniscript Introducción (2020-02-06)
+transcript_by: Michael Folkson
+translation_by: Blue Moon
+categories: ['conferencia']
+tags: ['miniscript', 'cartera']
+---
+
+Nombre: Andrew Poelstra
+
+Tema: Introducción a Miniscript
+
+Locación: El avance de Bitcoin
+
+Día: 6 de febrero de 2020 
+
+Vídeo: https://www.youtube.com/watch?v=eTUuwASdUBE
+
+Diapositivas: https://www.dropbox.com/s/vgh5vaooqqbgg1v/andrew-poelstra.pdf
+
+# Introducción
+
+Hola a todos. Algunos de vosotros estuvisteis en mi [presentación de Bitcoin](https://diyhpl.us/wiki/transcripts/london-bitcoin-devs/2020-02-04-andrew-poelstra-miniscript/) sobre Miniscript hace un par de días donde me las arreglé para pasar más de 2 horas dando una presentación sobre esto. Creo que esta vez la he reducido a 20 minutos, pero no prometo nada. Voy a mantener un ojo en el reloj.
```

Comment:
Please substitute with btctranscript.com links instead of diyhpl.us in the places they appear.